Update README2.md (#230)
githuba11y/repo_the_second2: README2.md

# repo_the_second2

> This repo, repo_the_second2 is purely for accessibility testing.

Author: ```suggestion
This repo is purely for accessibility testing.
```

> This repo, repo_the_second_2, is purely for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_2,** is purely for accessibility testing.
```

> This repo, **repo_the_second_2** is purely for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_3** is purely for accessibility testing.
```

> This repo, **repo_the_second_3** is purely for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_4** is purely for accessibility testing.
```

> This repo, **repo_the_second_4** is purely for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_2** is purely for accessibility testing.
```

> This repo, **repo_the_second_2** is purely for accessibility testing.

Author: ```suggestion
This repo, repo_the_second2 is purely for accessibility testing.
```

> This repo, **repo_the_second_2,** is purely for accessibility testing.

Author: ```suggestion
This repo, _repo_the_second_2_ is **purely** for accessibility testing.
```

> This repo, _repo_the_second_2_ is **purely** for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_2** is **purely** for accessibility testing.
```

> This repo, **repo_the_second_2** is _purely_ for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_2** is purely for accessibility testing.
```

> This repo, **repo_the_second_2** is **purely** for accessibility testing.

Author: ```suggestion
This repo, **repo_the_second_2** is _purely_ for accessibility testing.
```

> It contains no real content, all content (discussions, issues, PRs, projects, comments, etc) is dummmy content for testing.

Author: ```suggestion

It contains no real content, all content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.
```

It contains no real content, all content (discussions, issues, PRs, projects, comments, etc) is dummmy content for testing.

> It contains no real content, all content (discussions, issues, PRs, Milestones, etc) is dummmy content for testing.

Author: Commit comment

> It contains no real content, all content (discussions, issues, PRs, Milestones, etc) is dummmy content for testing.

Author: Commit comment

Author: Hello

This repo is purely for accessibility testing.

> It contains no real content, all content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.

Author: ```suggestion
It contains no real content. All content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.
```

> It contains no real content. All content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.

Author: ```suggestion
It contains no real content. 

All content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.
```

This repo is purely for accessibility testing.

> It contains no real content, all content (discussions, issues, PRs, Milestones, etc) is dummmy content for testing.

Author: We should keep this line.

Author: Responding

Author: Test

> It contains no real content, all content (discussions, issues, PRs, Milestones, etc) is dummmy content for testing.

Author: <img width="76" alt="Screen Shot 2022-01-07 at 11 49 37 AM" src="https://user-images.githubusercontent.com/16447748/148599080-3d579f34-c422-48a7-b5a4-b9772b51cf9f.png">
This is dummy content

> Everything (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.

Author: ```suggestion
All content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.
```

> All content (discussions, issues, PRs, projects, comments, etc) is dummy content for testing.

Author: This is dummy content
<img width="113" alt="Screen shot of blue box" src="https://user-images.githubusercontent.com/16447748/148598581-77247c2b-15a9-4bc4-bb1b-20b7ddd98d6b.png">

Author: This is more more dummy content
<img width="113" alt="Screen shot of blue box" src="https://user-images.githubusercontent.com/16447748/148598581-77247c2b-15a9-4bc4-bb1b-20b7ddd98d6b.png">

Author: This is more more dummy content
<img width="113" alt="Screen shot of blue box" src="https://user-images.githubusercontent.com/16447748/148598581-77247c2b-15a9-4bc4-bb1b-20b7ddd98d6b.png">
